cms/management/commands/create_initial_data.py

```diff
@@ -10,9 +10,25 @@ class Command(BaseCommand):
 
     help = 'Create initial data by using factories.'
 
-    def collect_initial_data_functions(self):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--app-label',
+            dest='app_label',
+            help='Provide an app label to create app specific data (e.g. --app_label boxes)',
+            default=False,
+        )
+
+    def collect_initial_data_functions(self, app_label=None):
```

Comment:
Can `app_label` be a positional-only parameter now? We pass `app_label` to `collect_initial_data_functions()` in line 66 so it seems like we can make it a positional-only parameter.